HW3.cpp

```diff
@@ -0,0 +1,54 @@
+//
+//  HW3.cpp
+//  Codu
+//
+//  Created by Farida Eid on 6/6/17.
+//  Copyright © 2017 Farida Eid. All rights reserved.
+//
+
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include "Dictionary.h"
+#include "WordSearch.h"
+
+using namespace std;
+
+
+// loads the grid saved in the file "grid.txt" into the grid array
+void loadGrid(vector<vector<char>>* grid)
```

Comment:
Why vector<vector<char>> and not vector<string> ?
There are arguments both ways, no need to change it, just to be sure you consider the options.